src/_content/posts/djangocon-us-grants.md

```diff
@@ -0,0 +1,63 @@
+- Reimbursements are made a week after the conference, which typically lasts 4 to 5 weeks, to consolidate. Be patient, as the team is working tirelessly to get everyone sorted. In case of questions during that period, reach out to us via [email](mailto:grants@djangocon.us)
+
+The dates are subject to change based on the review process and the number of applications.
+
+
+Attending a conference can be costly, and our [Opportunity Grants](https://docs.google.com/forms/d/e/1FAIpQLSfuriCI4JtAEcTfLy9Qdk2h_DfmaGKE544svRfMJLCOu5eLDQ/viewform) aim to relieve some of that financial burden. We strive to support attendees based on their individual needs. While we may not be able to fulfil every request in full, we are committed to helping as many deserving individuals as possible experience the value of DjangoCon US.
+
+
+These grants can assist with expenses such as:
+- **Travel costs**: Transportation to Chicago.
+- **Ground travel**: Getting to the conference to where you are staying.
+- **Accommodation**: Lodging during the conference.
+- **Meals**: Ensuring you have nourishment throughout the conference.
+- **Visa fees**: If applicable for international attendees.
+- **Other relevant expenses**: We understand that individual circumstances vary.
+
+The application process is straightforward and conducted through a Google form. We want to make it easy for you to express interest in attending DjangoCon US.
+
+The [form](https://docs.google.com/forms/d/e/1FAIpQLSfuriCI4JtAEcTfLy9Qdk2h_DfmaGKE544svRfMJLCOu5eLDQ/viewform) collects the following information to help us understand your situation and needs:
+- **Contact Information**: To communicate decisions.
+- **Affiliation**: Company, organisation, or individual.
+- **Django Background**: Your experience, reasons for attending, Django usage, and community involvement (to gauge interest).
+- **Attendance History**: First-time attendee status.
+- **Speaking Status**: If you've applied to speak.
+- **First-Time Speaker**: If accepted, would this be your first Django-related conference talk?
+- **Your Situation**: To understand your background.
+- **Financial Needs**: Funding amount (excluding ticket) and a cost breakdown (flights, travel, lodging, meals, visa, etc.).
+- **Payment Information**: Confirmation of PayPal or Wise account (flexible if needed).
+
+In the review process, we offer all applicants equal opportunities, regardless of their role, whether as speakers, attendees, volunteers, or in other capacities.
+
+As an open-source framework, we value open source. That is why we use an open-source tool called [Grorg](https://github.com/andrewgodwin/grorg), which is available to the Opportunity Grants committee volunteers who review all applications. Applications are constantly reviewed by more than one person to avoid bias and are scored by the reviewers.  
```

Comment:
```suggestion
We use an open-source tool called [Grorg](https://github.com/andrewgodwin/grorg), which is available to the Opportunity Grants committee volunteers who review all applications. Applications are constantly reviewed by more than one person to avoid bias and are scored by the reviewers.  
```